sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -123,6 +124,24 @@ class DataFrameReader private[sql](sqlContext: SQLContext) extends Logging {
   }
 
   /**
+   * Loads input in as a [[DataFrame]], for data sources that support multiple paths.
+   * Only works if the source is a HadoopFsRelationProvider.
+   *
+   * @since 1.6.0
+   */
+  def load(paths: Seq[String]): DataFrame = {
```

Comment:
This should be just a single `load(paths: Array[String])` as in other parts of the API.